src/main.rs

```diff
@@ -185,7 +192,7 @@ fn main() {
                 true if matches.is_present("pretty-json") => VariableOutputMode::PrettyJsonMode,
                 _ => VariableOutputMode::PosixMode,
             };
-            if let Err(err) = hook::run(data, mode) {
+            if let Err(err) = hook::run(env::current_dir().unwrap(), data, mode) {
```

Comment:
Is there a reason not to support `--dir` here as well?

Reply:
Yes: I think we should add a global `-C/--chdir` flag that causes shadowenv to change directory completely before executing, but it feels beyond the scope of this PR.